rajawali/src/main/java/org/rajawali3d/loader/LoaderOBJ.java

```diff
@@ -94,8 +94,12 @@
     protected final String DIFFUSE_COLOR = "Kd";
     protected final String DIFFUSE_TEX_MAP = "map_Kd";
 
+    private boolean mNeedToRenameMtl = true;
```

Comment:
In this case i need to set it "true" in other constructors. Now it is always "true", except this constructor
public LoaderOBJ(Renderer renderer, String fileOnSDCard)